src/docs/testing.mdx

```diff
@@ -63,6 +63,12 @@ pytest tests/snuba/api/endpoints/test_organization_events_distribution.py::Organ
 
 # Run all tests in a file that match a substring
 pytest tests/snuba/api/endpoints/test_organization_events_distribution.py -k method_name
+
+# When running tests, Django rebuilds the database on every run, which can make your test startup time slow. To avoid this, you can use
+# the --reuse-db flag, so that the database will persist between runs. This should significantly improve your test startup time after
```

Comment:
Looks good. Minor thing, maybe add the ticks to `--reuse-db` flag to emphasize the flag.

Reply:
Good call!